R/tidy-upkeep.R

```diff
@@ -180,6 +180,8 @@ tidy_minimum_r_version <- function() {
 }
 
 lowercase_r <- function() {
-  path <- proj_path(c("R", "tests/testthat"))
-  length(fs::dir_ls(path, regexp = "[.]r$")) > 0
+  path <- proj_path(c("R", "tests"))
+  path <- path[fs::dir_exists(path)]
+  any(fs::path_ext(fs::dir_ls(path, recurse = TRUE)) == "r")
```

Comment:
Is it intentional to shift from `dir_ls()`ing `tests/testthat/` to `tests/`?

Re: `path <- path[fs::dir_exists(path)]`. I think I would find a good ol' `if() { ... }` easier to read.

Reply:
No worries - I probably should have pluralized `path(s)` to make it clearer! Thanks!